src/Fonts.scss

```diff
@@ -0,0 +1,102 @@
+//h1 will be nasa font
+// open sans for header 2, 3
+//roboto for p
+
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+.opensans {
+  font-family: "Open Sans", sans-serif;
+}
+
+@import url('http://fonts.cdnfonts.com/css/bambi');
+
+.bambi{
+    font-family:'Bambi Bold', sans-serif;
+}
+
+@import url('https://typodermicfonts.com/nasalization/');
+
+.nasafont{
+    font-family:''
+  
+}
+// @font-face {
```

Comment:
let's get rid of this as it's not used and we can think about putting it in later if we want to